sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/SymbolicExecution/ExplodedGraphTests.cs

```diff
@@ -935,5 +935,93 @@ public void Test(string s)
 
             explodedGraph.Walk();
         }
+
+        [TestMethod]
+        [TestCategory("Symbolic execution")]
+        public void ExplodedGraph_DefaultLiteral()
+        {
+            var testInput = "var i = default(int); int j = default; System.IO.File k = default;";
+            var method = ControlFlowGraphTest.CompileWithMethodBody(string.Format(TestInput, testInput), "Bar", out var semanticModel);
+            var methodSymbol = semanticModel.GetDeclaredSymbol(method);
+            var varDeclarators = method.DescendantNodes().OfType<VariableDeclaratorSyntax>();
+            var iSymbol = semanticModel.GetDeclaredSymbol(varDeclarators.First(d => d.Identifier.ToString() == "i"));
```

Comment:
I think that we could extract a method, for symbol retrieving, based on it's name, to reduce a bit the code duplication.